Libraries/Text/TextInput/Multiline/RCTMultilineTextInputView.m

```diff
@@ -106,7 +106,7 @@ - (void)setEnableFocusRing:(BOOL)enableFocusRing {
 - (void)scrollViewDidScroll:(RCTUIScrollView *)scrollView // TODO(macOS ISS#3536887)
 {
   RCTDirectEventBlock onScroll = self.onScroll;
-
+  [_scrollView setHasVerticalScroller:YES];
```

Comment:
`_scrollView` is only defined for macOS as per here:
https://github.com/microsoft/react-native-macos/blob/7c3073ae838983aa90d9828bfbb0a09b1d637616/Libraries/Text/TextInput/Multiline/RCTMultilineTextInputView.m#L16-L18

Reply:
Thanks for catching this! I'll add a macro where I reference _scrollView.